tests/integration/test_fim/conftest.py

```diff
@@ -80,9 +80,23 @@ def create_monitored_folders(test_folders):
         test_folders(list): List of folders to create and delete
     """
     for folder in test_folders:
-        if os.path.exists(folder):
-            delete_path_recursively(folder)
-        os.mkdir(folder)
+        os.mkdir(folder, mode=0o0777)
+
+    yield
+    for folder in test_folders:
+        delete_path_recursively(folder)
+
+
+@pytest.fixture(scope='module')
+def create_monitored_folders_module(test_folders):
+    """
+    Create the folders that will be monitored and delete them at the end.
+    Args:
```

Comment:
```suggestion
    
    Args:
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3627/commits/9dab97f8fb4961c331782d0d96cca6510ea1d041